.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: go
+sudo: false
 go:
   - 1.3
```

Comment:
What would you think to removing 1.6 as well, given that 1.9 is just about to be out.

Then I could replace this (which doesn't work yet)

```
script:
  - go test -v ./...
  - go version | grep -qE 'go1\.[789]\.|devel' && ./compileall.sh
```

with this

```
script:
  - go test -v ./...
  - ./compileall.sh
```

This is because `go tool dist list` which compileall.sh uses only came in go1.7